configure.ac

```diff
@@ -127,10 +127,10 @@ AC_ARG_WITH([sqlite],
   [use_sqlite=auto])
 
 AC_ARG_WITH([bdb],
-  [AS_HELP_STRING([--without-bdb],
-  [disable bdb wallet support (default is enabled if wallet is enabled)])],
+  [AS_HELP_STRING([--with-bdb],
```

Comment:
I don't think we should include such a change in this repo, we should only focus on adding support for building with qml. 

What we can do is just not program support for generating a `bdb` wallet with the new QML gui, but leave this as is. I think, as fanquake states, we should try not to diverge too much from the core repo.

Reply:
Ok, the commit dropped.